cli/args/mod.rs

```diff
@@ -165,14 +166,21 @@ pub fn ts_config_to_emit_options(
       "precompile" => (false, false, false, true),
       _ => (false, false, false, false),
     };
+  let source_map = if options.inline_source_map {
+    SourceMapOption::Inline
+  } else if options.source_map {
+    SourceMapOption::Separate
+  } else {
+    SourceMapOption::None
+  };
   deno_ast::EmitOptions {
     use_ts_decorators: options.experimental_decorators,
     use_decorators_proposal: !options.experimental_decorators,
     emit_metadata: options.emit_decorator_metadata,
     imports_not_used_as_values,
-    inline_source_map: options.inline_source_map,
     inline_sources: options.inline_sources,
-    source_map: options.source_map,
+    keep_comments: false,
```

Comment:
This is a size optimization, right?

Reply:
Yes. There isn't a need to pass comments to v8, from my understanding.